invokeai/backend/model_manager/load/model_cache/model_cache_default.py

```diff
@@ -292,7 +292,13 @@ def move_model_to_device(self, cache_entry: CacheRecord[AnyModel], target_device
                     for k, v in cache_entry.state_dict.items():
                         new_dict[k] = v.to(target_device, copy=True)
                     cache_entry.model.load_state_dict(new_dict, assign=True)
-            cache_entry.model.to(target_device)
+            try:
+                cache_entry.model.to(target_device)
+            except TypeError as e:
+                if "got an unexpected keyword argument 'non_blocking'" in str(e):
+                    cache_entry.model.to(target_device)
+                else:
+                    raise e
```

Comment:
nit: Now that we removed the use of non-blocking on main (https://github.com/invoke-ai/InvokeAI/pull/6624), we could remove this try-except.